etc/ci/buildbot_steps.yml

```diff
@@ -27,6 +27,14 @@ mac-nightly:
   - ./mach package --release
   - ./etc/ci/upload_nightly.sh mac
 
+linux-intermittent:
```

Comment:
CI doesn't read from this file yet, it's currently reading from `buildbot/master/files/config/steps.yml` file in the saltfs repo, which is where we'll need to make this change.

Also, let's name these `linux-rel-intermittent` and `mac-rel-intermittent` for consistency.